profiles/markaspot/themes/ntxuva/js/ntxuva.js

```diff
@@ -360,12 +362,11 @@ var lang_pt = {
         },
         init: function(){
             var _self = this;
-
             if (!$('body').hasClass('front'))
                 return;
 
             $.ajax({
-                url: '/georeport/v2/requests.xml?start_date=2016-01-01',
+                url: '/georeport/v2/requests.json?start_date='+defaultStartDate, //default to load requets for the last 6 months
```

Comment:
I suggest you remove the comment saying its for last 6 months. If we update the interval above and forget updating the comment here, then the comment will trick people coming back to this code in the future. I think it suffices to say that we're using the default data, whoever wants to know about it will look it up.